e-signet/1.8.0/eSignet-signup-mosipid-UI-Automation/EsignetUi-esqa2-2026-05-08-00-12-T-17-P-9-F-0-S-0-KI-8.html

```
@@ -0,0 +1,1915 @@
+      <tr class="event-row">
+        <td><span class="badge log info-bg">Info</span></td>
+        <td>12:00:11 AM</td>
+        <td>
+          Entered Identifier Value - Verified visibility<details><summary>Locator Details</summary><pre>[[RemoteWebDriver: chrome on windows (e77b9dac058879bab7df6d3f9692c15b02f23435)] -> id: phone]</pre></details>
+        </td>
+      </tr>
+      <tr class="event-row">
+        <td><span class="badge log info-bg">Info</span></td>
+        <td>12:00:12 AM</td>
+        <td>
+          Entered Identifier Value - Entered text: '108067677'<details><summary>Locator Details</summary><pre>[[RemoteWebDriver: chrome on windows (e77b9dac058879bab7df6d3f9692c15b02f23435)] -> id: phone]</pre></details>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)